cli/wizard/commands/list-squid-token/index.ts

```diff
@@ -177,6 +188,8 @@ function parseAsInterchainTokenConfig(
     remoteTokens: data.remoteTokens.map((token) => ({
```

Comment:
```suggestion
    chains: data.remoteTokens.map((token) => ({
```
Don't think we should distinguish origin chain from remote chains. We can keep the `originAxelarChainId` but the details should be in the [map](https://github.com/axelarnetwork/chains/blob/main/generate_unified_configs.py#L226)